### PR TITLE
New package: karchive-5.26.0

### PR DESCRIPTION
**Pull request by dutsky**

### CODE REVIEW

**Comment by asergi** [comment]
Please rebase.

**Comment by dutsky** [comment]
As you wish, Master :)

File: srcpkgs/karchive/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'karchive'
+pkgname=karchive
+version=5.26.0
+revision=1
+wrksrc=${pkgname}-${version}
```
Comment (Vaelatern) [inline srcpkgs/karchive/template]: Redundant here

**Comment by dutsky** [comment]
must be done

**Pull request merged**